package.json

```diff
@@ -58,7 +58,8 @@
     "tslint-config-airbnb": "^5.11.2",
     "ttypescript": "^1.5.10",
     "typescript": "^3.7.5",
-    "yargs": "^15.1.0"
+    "yargs": "^15.1.0",
+    "improved-yarn-audit": "^2.2.1"
```

Comment:
I would rather install it on the fly just before running audit in CI - you don't really need this any other time.

Reply:
Need it to run locally